src/execution_plan/ops/op_cond_var_len_traverse.c

```diff
@@ -73,21 +72,20 @@ OpBase *NewCondVarLenTraverseOp(const ExecutionPlan *plan, Graph *g, AlgebraicEx
 
 	_setupTraversedRelations(op, ae->edge);
 
-	// Set our Op operations
 	OpBase_Init((OpBase *)op, OPType_CONDITIONAL_VAR_LEN_TRAVERSE,
-				"Conditional Variable Length Traverse", NULL, CondVarLenTraverseConsume, CondVarLenTraverseReset,
-				CondVarLenTraverseToString, CondVarLenTraverseFree, plan);
-
+				"Conditional Variable Length Traverse", NULL, CondVarLenTraverseConsume,
+				CondVarLenTraverseReset, CondVarLenTraverseToString, CondVarLenTraverseFree, plan);
 	assert(OpBase_Aware((OpBase *)op, ae->src_node->alias, &op->srcNodeIdx));
 	op->destNodeIdx = OpBase_Modifies((OpBase *)op, ae->dest_node->alias);
+	op->edgesIdx = OpBase_Modifies((OpBase *)op, op->ae->edge->alias);
```

Comment:
Should we be calling `OpBase_Modifies` regardless if the edge hasn't been explicitly referenced: MATCH (a)-[*]->(b) RETURN a,b ?

Reply:
fixed